src/recommendations/customize.md

```diff
@@ -15,3 +30,72 @@ However, to ensure Magento can successfully collect metrics for the recommendati
 |`<a data-bind="attr : {'data-sku' : sku, 'data-unit-id'}"...</a>`| Collects click events. <br/>**Note:** If you add any anchor tags, you must include these attributes.|
 
 Refer to the [template overview]({{ site.baseurl }}/guides/v{{ site.version }}/frontend-dev-guide/templates/template-walkthrough.html) for more information about how to modify templates in Magento.
+
+## Recommendation unit placement
+
+When you [create a recommendation](https://docs.magento.com/m2/ee/user_guide/marketing/create-new-rec.html), you specify [where you want the recommendation unit to appear](https://docs.magento.com/m2/ee/user_guide/marketing/product-recommendations.html#productrecplacement) on your page. In the Admin, you can select to place the recommendations either above or below the main content. You can, however, customize that location by editing the `*.xml` files responsible for the layout. Magento generates these files when you create a recommendation.
+
+1. Change to the `layout` directory:
+
+    ```bash
+    cd `<your theme>/ProductRecommendationsLayout/layout`
+    ```
+
+    Within this directory you will see several `<page handles>.xml` files
```

Comment:
Delete this line. The previous revision made that page handles variable unnecessary.